src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -511,10 +511,22 @@ Domain<3> BinaryCompactObject::create_domain() const {
       std::move(boundary_conditions_all_blocks),
       {{"ObjectAExcisionSphere",
         ExcisionSphere<3>{object_A_.inner_radius,
-                          {{object_A_.x_coord, 0.0, 0.0}}}},
+                          {{object_A_.x_coord, 0.0, 0.0}},
```

Comment:
Maybe a small comment saying "Direction of faces given by [reference to either documentation or code]"